src/debug/seer-integration.js

```diff
@@ -75,6 +75,14 @@ export const layerEditListener = cb => {
   });
 };
 
+export const removeLayer = id => {
+  if (!window.__SEER_INITIALIZED__ || !id) {
```

Comment:
Depends I guess, I usually do this to remove as much nesting as I can. In this case it might be better. I can refactor once I add the initialize method